packages/@coorpacademy-player-services/src/test/fixtures/slides.js

```diff
@@ -13,12 +16,14 @@ const slideStore = reduce(
   slidesData
 );
 
-const findByChapter = chapterRef => filter({chapter_id: chapterRef}, slidesData);
+const findByChapter = (chapterRef: string) => filter({chapter_id: chapterRef}, slidesData);
 
 // eslint-disable-next-line import/prefer-default-export,require-await
-const findById = async id => {
+const findById = async (id: string): Promise<Slide> => {
   if (!slideStore.has(id)) throw new Error(`Slide ${id} not found`);
   const slide = slideStore.get(id);
+
+  // $FlowFixMe
```

Comment:
? can you add a description of "why you added a silent on it" ?

Reply:
✅ ( ` // $FlowFixMe (typing postponed for this section) ` )